dbt_style_guide.md

```diff
@@ -124,6 +260,7 @@ with
 my_data as (
 
     select * from {{ ref('my_data') }}
+    where not is_deleted

```

Comment:
I've never been a fan of using `where not [boolean]` or even `where [boolean]`- I find it much more confusing and takes me extra time to think through than just seeing `where is_deleted is false` or `where is_deleted is true`. Yes it's a few extra letters to type, but truly infinitely more clear to the reader. 

If it's a performance thing I'm not aware of, then fine, I'm not going to die on this point, but for a readability document I'd vote to change.

Reply:
I'm okay with changing this, 👍 from me. I'll wait for a few more to chime in here before making the changes!

Reply:
After looking at this a bit more, I'm going to leave this as is. We're not telling anyone to use the logic in this way, they are free to write `where is_deleted is false` or `where is_deleted = false`.